### PR TITLE
NNS1-2965 delete menu metrics update tvl

### PR DESCRIPTION
**Pull request by coskucinkilic**
# Motivation

Turn TVL into a Link display it in MenuItems footer

# Changes

Remove MenuMetrics, move transitions to MenuItems.
Move TVL to MenuItems
Update TVL styling
MenuItems footer margins updated to match with Figma more closely.

# Tests

Screenshots test are updated
New tests are added for TVL existence and link in MenuItems

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by coskucinkilic** [comment]
This PR is based on [this one](https://github.com/dfinity/nns-dapp/pull/5477) so please only review the new changes or wait until the other PR is merged.

**Comment by coskucinkilic** [comment]
Screenshot 2024-09-18 at 12 35 25
New e2e screenshots are the same, looking into why this is displayed if ENABLE_METRICS is false in CI. My current guess is it is not, need proof.

**Comment by dskloetd** [comment]
> This PR is based on https://github.com/dfinity/nns-dapp/pull/5477 so please only review the new changes or wait until the other PR is merged.

FYI, you can change the merge base of the PR to the branch of another PR in order to only show the difference between the PR. You do this by editing the title and then using the menu that appears below the title.

**Comment by coskucinkilic** [comment]
I generally try not to do that because I can forget, or in teams that anyone can merge the prs maybe they wouldn't be aware that i changed the base. But for this occasion until it is merged i will keep the base to another pr.

**Comment by dskloetd** [comment]
I'll wait for the issue to be resolved. Just wanted to point out it exists. I mostly use it to look at my own PRs before I send them out for review.

File: frontend/src/tests/page-objects/MenuItems.page-object.ts [hunk 1]
```diff
@@ -39,6 +39,17 @@ export class MenuItemsPo extends BasePageObject {
...
   }
 
+  getTotalValueLockedLinkPo(): LinkPo {
+    return LinkPo.under({
+      element: this.root,
+      testId: "total-value-locked-component",
+    });
+  }
+
+  getTotalValueLockedButtonLink(): Promise<string | null> {
```
Comment (mstrasinskis) [inline frontend/src/tests/page-objects/MenuItems.page-object.ts:49]: Probably it's not needed if `.getHref()` is used here (https://github.com/dfinity/nns-dapp/pull/5487/files#diff-cbcb534e9810752446478d48cde6975dc2bd6a4b16f303408596c92db313624cR208)

**Review by mstrasinskis — COMMENTED** [review]
Looks good, just a couple of questions.

File: frontend/src/lib/components/common/MenuItems.svelte [hunk 2]
```diff
@@ -115,22 +118,28 @@
   }
 
   .menu-footer {
-    display: none;
     flex-direction: column;
     gap: var(--padding);
     // To accomodate the 100% on-chain logo
     // if that logo changes please update this margin as well
-    margin: auto var(--padding-3x) var(--padding-8x) 0;
+    margin: auto var(--padding-3x) calc(var(--padding-8x) + var(--padding-1_5x))
```
[the remark this answers is not in the document]
Comment (coskucinkilic) [reply frontend/src/lib/components/common/MenuItems.svelte:125]: It is not so obvious here but this is my change as well from the previous PR, after I saw that not everything follows padding I have updated the gap at the bottom to better reflect Figma designs. 

Will add a line for this in description

File: frontend/src/lib/components/common/MenuItems.svelte [hunk 3]
```diff
@@ -115,22 +118,28 @@
...
-      transform: translate(-150%, 0);
+      transform: translate(-300%, 0);
+      transition:
+        transform linear var(--animation-time-short),
+        opacity linear calc(var(--animation-time-short) / 2);
     }
-    //Hide menu footer similar to surrounding elements
+    //Hide menu footer on short screens
+    display: none;
     @media (min-height: 654px) {
```
[the remark this answers is not in the document]
Comment (coskucinkilic) [reply frontend/src/lib/components/common/MenuItems.svelte:143]: It was already there there I didn't add or remove it, not aware of the requirement as it was from a time before I joined.

File: frontend/src/lib/components/common/MenuItems.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/common/MenuItems.svelte:125]: Thank you.

File: frontend/src/lib/components/common/MenuItems.svelte (hunk 3 above)
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/common/MenuItems.svelte:143]: True, sorry.

**Review by mstrasinskis — APPROVED** [review]
Thank you!

**Pull request merged**